shared/spotify-api/src/mockClient.ts

```diff
@@ -163,56 +163,72 @@ export class MockSpotifyClient implements SpotifyDataSource {
     id: string,
     params?: { limit?: number; offset?: number; include_groups?: string }
   ): Promise<Spotify.Object.Paginated<Spotify.Object.AlbumSimplified>> {
-    throw new Error('Mock not implemented.');
+    const albums = Object.entries(mocks.albums).filter(
+      ([id, album]) => id == id
+    );
+    return {
+      total: albums.length,
+      limit: params?.limit ?? 10,
+      offset: params?.offset ?? 0,
+      href: 'http://mocked.com/artistAlbums',
+      next: 'http://mocked.com/artistAlbums',
+      previous: 'http://mocked.com/artistAlbums',
+      items: albums.values() as any,
```

Comment:
I don't think this quite works because `Object.entries` turns an object into a 2d array. Instead you probably want to `map` and get the value:

```suggestion
      items: albums.map(([, album]) => album)
```